tests/contrib/operators/test_ssh_operator.py

```diff
@@ -70,6 +70,24 @@ def test_hook_created_correctly(self):
         self.assertEqual(TIMEOUT, task.ssh_hook.timeout)
         self.assertEqual(SSH_ID, task.ssh_hook.ssh_conn_id)
 
+    def test_empty_command_is_none(self):
+        from airflow.exceptions import AirflowException
+        TIMEOUT = 20
+        COMMAND = None
+        task = SSHOperator(
+            task_id="test",
+            command=COMMAND,
+            dag=self.dag,
+            timeout=TIMEOUT,
+            ssh_conn_id="ssh_default"
+        )
+        self.assertIsNone(task.command)
+        if six.PY2:
```

Comment:
We support only Python 3.

Reply:
@mik-laj this PR is to the `v1-10-stable` branch which still supports Python 2.
See https://github.com/apache/airflow/blob/v1-10-stable/setup.py#L604

Reply:
@retornam I don't think we should create patch to stable branch, we should PR to master, and committer will cherry-pick to stable if they want to release this patch if possible.

Reply:
@zhongjiajie  There are two issues that prevent a clean cherrypick of this commit from `master` to the `v1-10-stable` branch. 

-  On the  `master` branch the `ssh_operator` has been deprecated in favor of `airflow.providers.ssh.operators.ssh`
- Python `2` support has been dropped in the `master` branch also.

The master branch right now is primarily catering to a future major release and since there are teams actively running `v10` right now, their interests will be better served by having this PR merged directly to that branch.

Let me know if you have any suggestions on how to better go about this.

Reply:
@ashb thoughts on this?

Reply:
I think we should change `airflow.providers.ssh.operators.ssh` instead of `airflow/contrib/operators/ssh_operator.py` and create patch to master, maybe we could including the commit in the next release.

Reply:
will cherry-pick to `v10-test` branch and them to `v10-stable` is possible and ready to release.

Reply:
I help you to ask some to review in slack, hope get some feedback soon.

Reply:
And anyway, in this specific case there is a dedicated way in six to do it:

```python
            six.assertRaisesRegex(AirflowException,
                                  "SSH command not specified. Aborting.$",
                                  task.execute, None)
```